easybuild/easyconfigs/d/datalad/datalad-0.18.4-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,73 @@
+easyblock = 'PythonBundle'
+
+name = 'datalad'
+version = "0.18.4"
+
+homepage = 'https://www.datalad.org/'
+description = "DataLad is a free and open source distributed data management system that keeps track of your data, \
+creates structure, ensures reproducibility, supports collaboration, \
+and integrates with widely used data infrastructure."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils', '2.39')
+]
+
+dependencies = [
+    ('git-annex', '10.20230802'),
+    ('Python', '3.10.8'),
+    ('tqdm', '4.64.1'),
+    ('PLY', '3.11'),
+    ('Rust', '1.65.0'),
```

Comment:
why is Rust needed as a dependency? if needed to build some of the python package, it should be a builddep.